### PR TITLE
[estuary] skin sync

### PR DESCRIPTION
**Pull request by BigNoid**
adds the following:
- Show channel number, channel name, episode name in second row of topbar, when playing live tv or per recordings by ksooo
- MyPVRRecordings: Use Listitem.ActualIcon to ensure channel logo is displayed by ksooo
- Fix for seeklabel by ronie
- Remove media source from estuary settings window by jjd-uk
- Adds Kodi repository as widget by HitcherUK
- Fix timeseek label by Voyager1
- Drop Player.SeekOffset

### CODE REVIEW

File: addons/skin.estuary/1080i/Home.xml [hunk 1]
```diff
@@ -449,15 +455,19 @@
...
-							<param name="image_path" value="special://skin/extras/home-images/add-ons.jpg"/>
-							<param name="text_label" value="$LOCALIZE[31119]" />
-							<param name="button_label" value="$LOCALIZE[31118]" />
-							<param name="button_onclick" value="ActivateWindow(addonbrowser)"/>
-							<param name="button_id" value="8600"/>
-							<param name="onup_id" value="8100"/>
-							<param name="visible" value="!Integer.IsGreater(Container(8100).NumItems,0) + !Integer.IsGreater(Container(8200).NumItems,0) + !Integer.IsGreater(Container(8300).NumItems,0) + !Integer.IsGreater(Container(8400).NumItems,0) + !Integer.IsGreater(Container(8500).NumItems,0)"/>
-							<param name="button2_onclick" value="Skin.SetBool(HomeMenuNoProgramsButton)"/>
+						<include content="WidgetListSquare">
+							<param name="content_path" value="addons://repository.xbmc.org/"/>
```
Comment (tamland) [inline addons/skin.estuary/1080i/Home.xml]: This is an internal path, don't hardcode it in the skin. You can use `addons://all/`

Comment (BigNoid) [inline addons/skin.estuary/1080i/Home.xml]: The idea was to only show the add ons from our repo ```addons://all/``` shows everything.

Comment (tamland) [inline addons/skin.estuary/1080i/Home.xml]: Why would you *want* it to be hardcoded to one repository?

Comment (BigNoid) [inline addons/skin.estuary/1080i/Home.xml]: I don't have a strong opinion on this. ping @HitcherUK

Comment (HitcherUK) [inline addons/skin.estuary/1080i/Home.xml]: The idea was to promote Kodi addons _only_ hence the heading 'Kodi Add-on Repository', all others are available through the 'Add-on browser' item at the top the screen.

**Comment by wiromare** [comment]
sorry but this fix for the seeklabel has more problem then before... if you click with the mouse on the seek bar the label seeking remains visible even if you pause and unpause

the labels fast forward and rewind are shown only for a bit, then reappears the seeking label

my PS #10245 it's always there :) 

@BigNoid

File: addons/skin.estuary/1080i/Variables.xml [hunk 2]
```diff
@@ -193,10 +193,10 @@
 		<value condition="Player.Forwarding32x | Player.Rewinding32x">32x</value>
 	</variable>
 	<variable name="SeekLabel">
-		<value condition="[Player.Paused + !Player.Caching] + !Player.Seeking + !Player.DisplayAfterSeek">$LOCALIZE[112]</value>
-		<value condition="!String.IsEmpty(Player.SeekStepSize) + ![player.forwarding | player.rewinding]">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekStepSize][/COLOR]</value>
-		<value condition="Player.DisplayAfterSeek + ![player.forwarding | player.rewinding]">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekOffset][/COLOR]</value>
-		<value condition="!Player.DisplayAfterSeek + Player.Seeking">$LOCALIZE[773]</value>
+		<value condition="!String.IsEmpty(Player.SeekStepSize)">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekStepSize][/COLOR]</value>
+		<value condition="!String.IsEmpty(Player.SeekOffset)">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekOffset][/COLOR]</value>
+		<value condition="Player.Caching">$LOCALIZE[15107]</value>
```
Comment (FernetMenta) [inline addons/skin.estuary/1080i/Variables.xml]: I will drop this. caching is useless to know for the user. btw. 15107 seems to to exist.

File: addons/skin.estuary/1080i/Variables.xml [hunk 3]
```diff
@@ -193,10 +193,10 @@
 		<value condition="Player.Forwarding32x | Player.Rewinding32x">32x</value>
 	</variable>
 	<variable name="SeekLabel">
-		<value condition="[Player.Paused + !Player.Caching] + !Player.Seeking + !Player.DisplayAfterSeek">$LOCALIZE[112]</value>
-		<value condition="!String.IsEmpty(Player.SeekStepSize) + ![player.forwarding | player.rewinding]">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekStepSize][/COLOR]</value>
-		<value condition="Player.DisplayAfterSeek + ![player.forwarding | player.rewinding]">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekOffset][/COLOR]</value>
-		<value condition="!Player.DisplayAfterSeek + Player.Seeking">$LOCALIZE[773]</value>
+		<value condition="!String.IsEmpty(Player.SeekStepSize)">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekStepSize][/COLOR]</value>
+		<value condition="!String.IsEmpty(Player.SeekOffset)">$LOCALIZE[773][COLOR=grey] $INFO[Player.SeekOffset][/COLOR]</value>
```
Comment (FernetMenta) [inline addons/skin.estuary/1080i/Variables.xml]: drop SeekOffset, this label will disappear

File: addons/skin.estuary/1080i/Variables.xml (hunk 2 above)
Comment (FernetMenta) [inline addons/skin.estuary/1080i/Variables.xml]: disregard my comment regarding caching. sorry for confusion.

**Comment by ronie** [comment]
@wiromare those issues should be fixed now (dropping the seekoffset label).

**Pull request merged**